package.json

```diff
@@ -40,7 +40,7 @@
     "mutant-pull-reduce": "^1.1.0",
     "obv": "0.0.1",
     "patch-settings": "^1.0.1",
-    "patchcore": "~1.13.2",
+    "patchcore": "~1.13.3",
```

Comment:
What I normally do in this case is reference the git repo and commit directly in the package json.

If you do `npm install Happy0/patchcore#canRender --save` it will update the entry in the package.json to:

```
"patchcore": "github:Happy0/patchcore#canRender"
``` 

and will use the version directly off github. The only downside is that if you delete the branch, you can no longer install that particular fork.

Reply:
Oh, cool! I didn't know you could do that :). Done.

Goodnight!